packages/babel-helper-transform-fixture-test-runner/src/index.ts

```diff
@@ -381,11 +386,20 @@ expect.extend({
   },
 });
 
+declare global {
+  // eslint-disable-next-line no-redeclare
+  namespace jest {
+    interface Matchers<R> {
+      toEqualFile({ filename, code }): jest.CustomMatcherResult;
```

Comment:
It seems to me this is a hotfix around `@types/jest`, do we have an upstream issue to track here?

Reply:
It's because we implement `toEqualFile`, it's not defined by Jest itself.